crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -338,7 +357,16 @@ module deepbook::clob_v2 {
 
         let pool_uid = object::new(ctx);
         let pool_id = *object::uid_as_inner(&pool_uid);
-    
+
+        // Creates the capability to mark a pool owner.
+        let id = object::new(ctx);
+        let owner = object::uid_to_address(&id);
+        let pool_owner_cap = PoolOwnerCap { 
+            id, 
+            owner
+        };
+        transfer::public_transfer(pool_owner_cap, tx_context::sender(ctx));
```

Comment:
I would suggest:
- returning the `PoolOwnerCap` from this function
- adding the transfer to sender inside the existing callers
- Adding a new function that gives the caller both a `Pool` and the `PoolOwnerCap` (for better PTB-friendliness/compositionally--e.g., if someone wants to create a pool and immediately destroy the owner cap or lock it in a smart contract)

Reply:
I'm ok with adding a new function, but because I see a lot of duplicate functionality in `customizable_pools` and `default_pools`, I will only add a new function that has the customizable attributes.